### PR TITLE
Review of Flash.md

### PR DESCRIPTION
**Pull request by sabeena21**
### Change Description ###

Made minor modifications to the original content.

### CODE REVIEW

File: docs/application/tizen-studio/rt-ide/getting-started/flash.md [hunk 1]
```diff
@@ -2,38 +2,32 @@
...
      ![Flashing the project](media/rt_flash.png)
 
-   If you have not built the project yet, an error popup appears.
+   If you have not built the project yet, an error popup appears:
 
-   ![Flashing without building error](media/rt_flash_build_error.png)
+     ![Flashing without building error](media/rt_flash_build_error.png)
 
-4. In the Flash TizenRT Project Wizard, select the Flash option and click **Flash**.
+4. In the Flash TizenRT Project Wizard, select the **Flash** option, and click **Flash**:
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/getting-started/flash.md]: The description and the image doesn't match.

>
> image
>

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/483

File: docs/application/tizen-studio/rt-ide/getting-started/flash.md [hunk 2]
```diff
@@ -2,38 +2,32 @@
...
      ![Flashing the project](media/rt_flash.png)
 
-   If you have not built the project yet, an error popup appears.
+   If you have not built the project yet, an error popup appears:
 
-   ![Flashing without building error](media/rt_flash_build_error.png)
+     ![Flashing without building error](media/rt_flash_build_error.png)
 
-4. In the Flash TizenRT Project Wizard, select the Flash option and click **Flash**.
+4. In the Flash TizenRT Project Wizard, select the **Flash** option, and click **OK**:
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/getting-started/flash.md]: Question. Is this update checked with the actual Tizen Studio? Why the bold font is applied to the **Flash**?

Comment (sabeena21) [reply docs/application/tizen-studio/rt-ide/getting-started/flash.md]: @mijin85cho, Thank you for pointing it out. I will remove bold tag.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/483

**Pull request merged**